app/src/main/java/com/antosik/benchproject/app/common/BenchBottomNavigationView.kt

```diff
@@ -0,0 +1,53 @@
+package com.antosik.benchproject.app.common
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.animation.TranslateAnimation
+import androidx.core.view.isVisible
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class BenchBottomNavigationView(context: Context, attrs: AttributeSet) : BottomNavigationView(context, attrs) {
+
+    private val fromXDelta = 0f
+    private val toXDelta = 0f
+    private val fromYDelta = 0f
+    private val toYDelta = 0f
```

Comment:
Maybe create some private object (nested in this class) and keep these values there to have a better naming as these are only connected with the animation.

Reply:
I changed it